### PR TITLE
Add Input.Password

### PR DESCRIPTION
**Pull request by orzyyyy**
For Issue #13267.

### CODE REVIEW

**Comment by ilanus** [comment]
Great job! we are missing `eye-close` type of icon

hswm

Maybe use `show / hide` text instead of icon?

File: components/input/index.tsx [hunk 1]
```diff
@@ -2,13 +2,16 @@ import Input from './Input';
 import Group from './Group';
 import Search from './Search';
 import TextArea from './TextArea';
+import Password from './Password';
 
 export { InputProps } from './Input';
 export { GroupProps } from './Group';
 export { SearchProps } from './Search';
 export { TextAreaProps } from './TextArea';
+export { PasswordProps } from './Password'
```
Comment (ilanus) [inline components/input/index.tsx]: `;`

File: components/input/Input.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import omit from 'omit.js';
 import Group from './Group';
 import Search from './Search';
 import TextArea from './TextArea';
+import Password from './Password'
```
Comment (ilanus) [inline components/input/Input.tsx]: `;`

**Comment by zy410419243** [comment]
@ilanus Thanks for review and advice. I would have some try as below,
  
- add `Button` that the text can be customed by user.
  
- when click `Button` or `Icon`, the show of text should have a transition.

**Comment by zy410419243** [comment]
#13281

**Pull request closed without merging**